packages/console/src/pages/TenantSettings/TenantMembers/Invitations/index.tsx

```diff
@@ -0,0 +1,152 @@
+import { OrganizationInvitationStatus } from '@logto/schemas';
+import { format } from 'date-fns';
+import { useContext, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import useSWR from 'swr';
+
+import Plus from '@/assets/icons/plus.svg';
+import UsersEmptyDark from '@/assets/images/users-empty-dark.svg';
+import UsersEmpty from '@/assets/images/users-empty.svg';
+import { useAuthedCloudApi } from '@/cloud/hooks/use-cloud-api';
+import { type TenantInvitationResponse } from '@/cloud/types/router';
+import ActionsButton from '@/components/ActionsButton';
+import { RoleOption } from '@/components/OrganizationRolesSelect';
+import { TenantsContext } from '@/contexts/TenantsProvider';
+import Button from '@/ds-components/Button';
+import Table from '@/ds-components/Table';
+import TablePlaceholder from '@/ds-components/Table/TablePlaceholder';
+import Tag from '@/ds-components/Tag';
+import { type RequestError } from '@/hooks/use-api';
+
+const convertInvitationStatusToTagStatus = (status: OrganizationInvitationStatus) => {
+  switch (status) {
+    case OrganizationInvitationStatus.Pending: {
+      return 'alert';
```

Comment:
better assert the return type of this method

Reply:
ok

Reply:
updated